workspaces/tech-insights/plugins/tech-insights-react/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@backstage-community/plugin-tech-insights-react",
+  "description": "Web library for the tech-insights plugin",
+  "version": "0.1.0",
```

Comment:
Let's start with a major. It's bad habits to keep things in the exception branch of semver for so long. Significantly better for end users to use the lib with standard semver.